tests/system/libraries/WindowsLib.py

```diff
@@ -119,12 +121,18 @@ def taskSwitchToItemMatching(targetWindowNamePattern: _re.Pattern, maxWindowsToT
 		spy.emulateKeyPress('rightArrow')  # move to the next task switcher item
 		spy.wait_for_speech_to_finish(speechStartedIndex=nextIndex)
 		speech = spy.get_speech_at_index_until_now(nextIndex)
-		builtIn.log(f"next window: {speech}", level="DEBUG")
+		builtIn.log(f"next window: '{speech}'", level="DEBUG")
 		if targetWindowNamePattern.search(speech):
+			builtIn.log(f"Found target window at item {windowsTested}", level="DEBUG")

```

Comment:
```suggestion
			builtIn.log(f"Found target window at item '{windowsTested}'", level="DEBUG")
```

Reply:
Since `windowsTested` is an integer I'm not sure that quotes clarifies anything here?